ui/analyse/src/study/multiBoard.ts

```diff
@@ -135,60 +135,66 @@ function pagerButton(text: string, icon: string, click: () => void, enable: bool
   });
 }
 
-function makePreview(study: StudyCtrl) {
+const makePreview = (study: StudyCtrl) => {
   return (preview: ChapterPreview) => {
-    const contents = preview.players
-      ? [makePlayer(preview, opposite(preview.orientation)), makeCg(preview), makePlayer(preview, preview.orientation)]
-      : [h('div.name', preview.name), makeCg(preview)];
     return h(
-      'a.' + preview.id,
+      `a.${preview.id}.mini-game.mini-game-${preview.id}.mini-game--init.is2d`,
       {
-        attrs: { title: preview.name },
+        attrs: {
+          'data-state': `${preview.fen},${preview.orientation},${preview.lastMove}`,
+          'data-live': preview.id,
+        },
         class: {
           active: !study.multiBoard.loading && study.vm.chapterId == preview.id && !study.relay?.tourShow.active,
         },
-        hook: bind('mousedown', _ => study.setChapter(preview.id)),
+        hook: {
+          insert(vnode) {
+            const el = vnode.elm as HTMLElement;
+            lichess.miniGame.init(el);
+            vnode.data!.fen = preview.fen;
+            el.addEventListener('mousedown', _ => study.setChapter(preview.id));
+          },
```

Comment:
Couldn't find how to use `bind('mousedown', _ => study.setChapter(preview.id))` when also defining another hook

Reply:
You can't use `bind` if you want to add more to the `insert` hook. Otherwise, you could do:

```js
{
    hooks: {
        ...bind(...),
        otherHook: () => {},
    }
}
```